extensions/desktop/kde-neon-6/launcher-specific

```diff
@@ -52,7 +52,17 @@ echo "$SNAP/usr/lib/$ARCH" >> "$XDG_CONFIG_HOME/qtchooser/default.conf"
 # 0001-let-qlibraryinfo-fall-back-to-locate-qt.conf-via-XDG.patch
 # to make QLibraryInfo look in XDG_* locations for qt.conf. The paths configured
 # here are applied to everything that uses QLibraryInfo as final fallback and
-# has no XDG_* fallback before that. Currently the most interesting offender
+# has no XDG_* fallback before that. Currently the most interesting of
+export VLC_PLUGIN_PATH="$SNAP_DESKTOP_RUNTIME/usr/lib/$ARCH/vlc/plugins"
```

Comment:
Was this new section meant to go above line 51?

Reply:
No and I am sure a kitten was involved. Will fix here shortly.